contracts/core/RevenuePath.sol

```diff
@@ -603,19 +592,21 @@ contract RevenuePath is Ownable, Initializable, ReentrancyGuard {
             revert InsufficientWithdrawalBalance();
         }
 
+        uint256 thisRelease;
         uint256 payment = ethRevenuePending[account];
+        thisRelease = payment;
         released[account] += payment;
-        totalReleased += payment;
         ethRevenuePending[account] = 0;
 
         if (feeAccumulated > 0) {
             uint256 value = feeAccumulated;
             feeAccumulated = 0;
-            totalReleased += value;
-            platformFeeWallet = IReveelMain(mainFactory).getPlatformWallet();
-            sendValue(payable(platformFeeWallet), value);
+            thisRelease += value;
+            sendValue(payable(IReveelMain(mainFactory).getPlatformWallet()), value);
```

Comment:
I had broken it down, more for code readability reasons. But this also does the same thing. :+1: